kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/converter/altoida/AltoidaMetadataConverter.kt

```diff
@@ -44,11 +40,10 @@ class AltoidaMetadataConverter(override val sourceType: String = "altoida_metada
                 return listOf(convertedLine)
             }
         } catch (exe: IOException) {
-            log(LogLevel.WARN,"Something went wrong while processing contents of file ${contents.fileName}: ${exe.message} ")
+//            log(LogLevel.WARN,"Something went wrong while processing contents of file ${contents.fileName}: ${exe.message} ")
```

Comment:
Can this work again?

Reply:
Not yet. There is a simple log-repository in the "RecordConverter". Now that I have separated the `DataProcessor` from it, I have to find a way to share this log-repository with the processors too.  Again I want to make it a separate Pull request and maybe want to rethink how we store the logs of each record in the back-end.